apps/api-extractor-model/src/model/ApiEntryPoint.ts

```diff
@@ -51,4 +53,23 @@ export class ApiEntryPoint extends ApiItemContainerMixin(ApiNameMixin(ApiItem))
     // No prefix needed, because ApiEntryPoint is the only possible member of an ApiPackage
     return this.name;
   }
+
+  /** @beta @override */
+  public buildCanonicalReference(): DeclarationReference {
+    let modulePath: string = '';
+
+    if (this.parent instanceof ApiPackage) {
+      modulePath = this.parent.name;
+    }
+
+    if (this.name) {
+      modulePath += this.name;
```

Comment:
Note:  API Extractor treats the _NPM package name_ and _import path_ as two separate components, so we have to reassemble them here.

Reply:
Wouldn't that just be something like `packageName + '/' + packageRelativeImportPath`?

In other words:

- `foo` - package: '"foo"', import path: `""` 
- `foo/bar` - package: "foo", import path: `"bar"`
- `@scopes/foo/bar` - package: `"@scoped/foo"`, import path: `"bar"` ?

Reply:
Yes, that's what I'm doing in this code.

Reply:
Is `this.name` guaranteed to have a `/` prefix?

Reply:
Heh, the spec is fairly vague.  Although `ApiEntryPoint` anticipates future support for multiple entry points, it's always `""` today.

I'll clarify this by adding a property like `moduleFullPath` and documenting what kind of string would be expected to appear in `ApiEntryPoint.name` in the future.